nextjs/node_modules/fs-capacitor/lib/index.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
```

Comment:
Any reason `fs-capacitor` should be version controlled 🤔 ?

Reply:
Weird! Not sure how that happened thanks 👍